README.md

```diff
@@ -1,17 +1,44 @@
-# Google CloudEvents – Python
+# Google Events Library for Python
 
-This repository contains types for CloudEvents issued by Google.
+This library provides classes of common event types on Google Cloud Platform
```

Comment:
We don't want to limit this to GCP. Say Google Cloud which includes G Suite, Firebase, etc.

Reply:
Done.